numpy/linalg/lapack_litemodule.c

```diff
@@ -17,6 +17,8 @@ typedef struct { float r, i; } f2c_complex;
 typedef struct { double r, i; } f2c_doublecomplex;
 /* typedef long int (*L_fp)(); */
 
+void FNAME(MAIN_)() { return 0; }
```

Comment:
A `void` function returns nothing, not `0`. Also, you need to specify that there aren't any parameters to the function with `void`, the empty parentheses mean that in C++, but not in C. Lastly, the `FNAME` macro will append an underscore to the name, are you sure it is `FNAME(MAIN_)` and not `FNAME(MAIN)` what you want?

In short, I think replacing this line with

```C
void FNAME(MAIN)(void) {}
```

will take care of the compiler warnings making Travis fail.